aerosandbox/library/power_solar.py

```diff
@@ -120,6 +195,55 @@ def scattering_factor(elevation_angle):
     # scattering_factor = cas.fmin(cas.fmax(scattering_factor, 0), 1)
     return scattering_factor
 
+def scattering_factor_vert(elevation_angle):
```

Comment:
Is this the correct implementation here? Is the scattering that we are accounting for atmospheric scattering (in which case panel orientation is unimportant) or scattering/reflection within the panel itself (in which case panel orientation is important). My interpretation of the scattering chart was the former, but we might want to double-check this.